R/get.R

```diff
@@ -4,14 +4,19 @@
 
 ### called by getConquestPVs ---------------------------------------------------
 
-get.plausible <- function(file, quiet = FALSE, forConquestResults = FALSE)  {
+get.plausible <- function(file, quiet = FALSE, forConquestResults = FALSE){
+  checkmate::assert_file(file)
```

Comment:
Should I use `assert_file` or `assert_character` for asserting the `file` argument?

Reply:
I think, `assert_file()` is appropriate here.